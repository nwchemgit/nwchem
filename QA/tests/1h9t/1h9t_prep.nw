echo
start md_dat

prepare
  print sequence
  system 1h9t_md
  new_rst
  new_top new_seq
  source start_md_neutral.pdb
  #solvent name HOH model spce
  #solvate 
  vacuo
end

task prepare
