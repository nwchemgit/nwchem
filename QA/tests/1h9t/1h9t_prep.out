 argument  1 = 1h9t_prep.nw



============================== echo of input deck ==============================
echo
start md_dat

prepare
  system 1h9t_md
  new_rst
  new_top new_seq
  source start_md_final.pdb
  #solvent name HOH model spce
  #solvate
  vacuo
end

task prepare
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.8
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2018
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = hpc1.csc.bnl.gov
    program         = ../../../bin/LINUX64/nwchem
    date            = Wed Aug  8 18:28:02 2018

    compiled        = Wed_Aug_08_18:27:54_2018
    source          = /home/hvandam/nwchem-1-codar
    nwchem branch   = Development
    nwchem revision = nwchem_on_git-509-g452f955
    ga revision     = 5.6.5
    use scalapack   = F
    input           = 1h9t_prep.nw
    prefix          = md_dat.
    data base       = ./md_dat.db
    status          = startup
    nproc           =        1
    time left       =      0s



           Memory information
           ------------------

    heap     =     217724 doubles =      1.7 Mbytes
    stack    =  178884395 doubles =   1364.8 Mbytes
    global   =  357768789 doubles =   2729.6 Mbytes (distinct from heap & stack)
    total    =  536870908 doubles =   4096.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                  Prepare Module
                                  --------------

 Force field                           amber


 Directories used for fragment and segment files

                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_s/
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_x/
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_q/
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_t/
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_u/
                                       ./

 Parameter files used to resolve force field parameters

                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_s/amber.par
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_x/amber.par
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_q/amber.par
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_t/amber.par
                                       /home/hvandam/nwchem-1-codar/QA/../src/data/amber_u/amber.par
                                       ./amber.par
 PDB geometry                          start_md_final.pdb


 Creating fragment for residue PRO_M

 Atom type definitions                 /home/hvandam/nwchem-1-codar/QA/../src/data/amber_s/amber.par
 Atom type definitions                 /home/hvandam/nwchem-1-codar/QA/../src/data/amber_x/amber.par
 Atom type definitions                 /home/hvandam/nwchem-1-codar/QA/../src/data/amber_q/amber.par
 Atom type definitions                 /home/hvandam/nwchem-1-codar/QA/../src/data/amber_t/amber.par
 Atom type definitions                 /home/hvandam/nwchem-1-codar/QA/../src/data/amber_u/amber.par
 Bond charge definitions               /home/hvandam/nwchem-1-codar/QA/../src/data/amber_s/amber.par
 Bond charge definitions               /home/hvandam/nwchem-1-codar/QA/../src/data/amber_x/amber.par
 Bond charge definitions               /home/hvandam/nwchem-1-codar/QA/../src/data/amber_q/amber.par
 Bond charge definitions               /home/hvandam/nwchem-1-codar/QA/../src/data/amber_t/amber.par
 Bond charge definitions               /home/hvandam/nwchem-1-codar/QA/../src/data/amber_u/amber.par

 Fragment                              PRO_M

  num name  type   link cntr  grp pgrp    charge     polarizab

    1  N              0    0    0    1    1    0.000000    0.000000
    2  CD   CT        0    0    0    1    1   -0.100000    0.000000
    3 3HD   H1        0    0    0    1    1    0.050000    0.000000
    4 2HD   H1        0    0    0    1    1    0.050000    0.000000
    5  CG   CT        0    0    0    1    1   -0.100000    0.000000
    6 3HG   HC        0    0    0    1    1    0.050000    0.000000
    7 2HG   HC        0    0    0    1    1    0.050000    0.000000
    8  CB   CT        0    0    0    1    1   -0.100000    0.000000
    9 3HB   HC        0    0    0    1    1    0.050000    0.000000
   10 2HB   HC        0    0    0    1    1    0.050000    0.000000
   11  CA             0    0    0    1    1    0.000000    0.000000
   12  HA             0    0    0    1    1    0.000000    0.000000
   13  C              0    0    0    1    1    0.000000    0.000000
   14  O              0    0    0    1    1    0.000000    0.000000
                                           ------------
                         total charge     0.000000

 Connectivity

       1-  2
       2-  3
       2-  4
       2-  5
       5-  6
       5-  7
       5-  8
       8-  9
       8- 10
       8- 11
      11- 12
      13- 14
 
 Created fragment                      ./PRO_M.frg_TMP

 Unresolved atom types in fragment PRO_M


 **********
 *   0: pre_mkfrg failed 9999
 **********
